Makefile

```diff
@@ -124,4 +130,4 @@ test-clean:
 verify: check format test
 
 .PHONY: verify-extended
-verify-extended: install-requirements check-generate check format test-cov test-clean
```

Comment:
Can we entirely drop `install-requirements`?

Reply:
Hm, I think it is still useful for setup of local dev environment.
It has also not be removed from the Makefile of the provider extensions, e.g. [provider-aws](https://github.com/gardener/gardener-extension-provider-aws/blob/master/Makefile#L100)

Reply:
Okay, we should at least remove the double maintained tools like `gen-crd-api-reference-docs`, `mockgen`, `goimports`.
The point is that by including `$(REPO_ROOT)/vendor/github.com/gardener/gardener/hack/tools.mk`, tools are downloaded and used from `/vendor/github.com/gardener/gardener/hack/tools/bin`, i.e. `tools.mk` also determines the versions.
It can now happen that if a developer sets up his environment, he gets a different version via `make install-requirements` than the tools that are for instance used for `make verify-extended`.

Reply:
done